### ISSUE TITLE
Blazing Hippogryph cannot be used as a ground mount

### ISSUE THREAD

**Issue by arestavo** [dalaranwow/dalaran-wow#5920]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)
http://i.imgur.com/RFouVVi.png
**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Blazing Hippogryph cannot be summoned in Dalaran or anywhere else that a ground mount can be summoned.
**Expected behaviour**: Tell us what should happen instead
It should be able to be summoned as a ground mount anywhere a ground mount it summonable.
**Steps to reproduce the problem**:

1. Donate to server.
2. Use 14 donation points to get the Blazing Hippogryph.
3. Look with disappointment at your screen because you cannot summon it as a ground mount.

**Include proofs for this behaviour**
![WoWScrnShot_081520_113114](https://user-images.githubusercontent.com/50332138/90318106-6025eb00-deeb-11ea-8242-d9a52cc807f0.jpg)
![WoWScrnShot_081520_113123](https://user-images.githubusercontent.com/50332138/90318107-61571800-deeb-11ea-8759-d3e794924da5.jpg)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.
https://www.wowhead.com/item=54069/blazing-hippogryph
**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
https://www.wowhead.com/item=54069/blazing-hippogryph
You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by alexuktc** [comment]
Hello Arestavo. 

Upon further inspection of the mount it would seem that the mount indeed doesn't have a ground version in WotLK. It was added in patch 4.2, or so the comments on WoWHead say. 

https://www.wowhead.com/item=54069/blazing-hippogryph#comments Look for rowanr's comment. At least this is the only thing I found on the matter.

**Comment by Golganeth** [comment]
All sources that I've found, mention that it wasn't possible to use this mount as a ground one prior to patch 4.x, as @alexuktc mentioned.
Please @arestavo provide enough proof to backup your statement and we will look into this.

Closing
